Whiskey/Functions/Invoke-WhiskeyNodeTask.ps1

```diff
@@ -71,7 +73,10 @@ function Invoke-WhiskeyNodeTask
         }
         return
     }
-
+    $npmRegistryUri = $TaskParameter['NpmRegistryUri']

```

Comment:
This parameter is required. It should always be provided. We don't want to use `ProGetSession.NpmFeedUri`.

Once you've made `NpmRegistryUri` mandatory (and fixed other issues and failing tests), remove the NpmFeedUri from ProGetSession and remove parameters from all functions that pass it